tests/integration/test_vulnerability_detector/test_scan_results/test_scan_nvd_feeds.py

```diff
@@ -73,6 +74,11 @@ def mock_vulnerability_scan(request):
     """
     It allows to mock the vulnerability scan inserting custom packages, feeds and changing the host system
     """
+    global vulnerabilities_number
+    if "RHEL" in request.param['target']:
+        vulnerabilities_number = 0
+    else:
+        vulnerabilities_number = len(nvd_vulnerabilities['vulnerabilities'])
```

Comment:
I think that this block can be improved. For instance, you can add the `vulnerabilities_number` variable to `system_data` and then use it in the fixture.